rfcs/SchemaCoordinates.md

```diff
@@ -0,0 +1,524 @@
+
+  This spec does _not_ support selecting schema members with a path from a root
+  type (e.g. `Query`).
+
+  For example, given this schema
+
+  ```graphql
+  type User {
+    name: String
+    bestFriend: User
+  }
+
+  type Query {
+    userById(id: String): User
+  }
+  ```
+
+  The following are invalid schema coordinates:
+
+  - `Query.userById.name`
+  - `User.bestFriend.bestFriend.bestFriend.name`
+
+  This violates a non-goal that there be one, unambiguous way to write a
+  schema coordinate to refer to a schema member. Both examples can be
+  "simplified" to `User.name`, which _is_ a valid schema coordinate.
+
+  Should a use case for this arise in the future, a follow up RFC may investigate
+  how schema coordinates could work with "field paths" (e.g. `["query", "searchBusinesses", 1, "owner", "name"]`) to cover this.
+
+- **Directive applications**
+
+  This spec does _not_ support selecting applications of directive.
+
+  For example:
+
+  ```graphql
+  directive @private(scope: String!) on FIELD
+
+  type User {
+    name: String
+    reviewCount: Int
+    friends: [User]
+    email: String @private(scope: "loggedIn")
+  }
+  ```
+
+  You _can_ select the definition of the `private` directive and its arguments
+  (with `@private` and `@private(scope:)` respectively), but you cannot select the
+  application of the `@private` on `User.email`.
+
+  For the stated use cases of this RFC, it is more likely that consumers want to
+  select and track usage and changes to the definition of the custom directive
+  instead.
+
+  If we _did_ want to support this, a syntax such as `User.email@private[0]`
+  could work. (The indexing is necessary since [multiple applications of the same
+  directive is allowed][multiple-directives], and each is considered unique.)
+
+  [multiple-directives]: http://spec.graphql.org/draft/#sec-Directives-Are-Unique-Per-Location
+
+- **Union members**
+
+  This spec does not support selecting members inside a union definition.
+
+  For example:
+
+  ```graphql
+  type Breakfast {
+    eggCount: Int
+  }
+
+  type Lunch {
+    sandwichFilling: String
+  }
+
+  union Meal = Breakfast | Lunch
+  ```
+
+  You may select the `Meal` definition (as "`Meal`"), but you may **not** select
+  members on `Meal` (e.g. `Meal.Breakfast` or `Meal.Lunch`).
+
+  It is unclear what the use case for this would be, so we won't (yet?) support
+  this. In such cases, consumers may select type members directly (e.g. `Lunch`).
+
+## 🤔 Drawbacks / Open questions
+
+- Should we specify an algorithm for doing the query -> set of schema
+  coordinates? Or just hint/imply that this mapping theoretically exists? Is this
+  out of scope?
+
+### Answered questions
+
+- **Is this extensible enough?** The above issue discusses adding arguments as
+  part of this specification - we haven't touched on this here in order to keep
+  this RFC small, but we may wish to consider this in the future (e.g.
+  `Query.searchBusiness:name`).
```

Comment:
Let's strikethrough this since the update below notes it's no longer relevant. Alternatively remove this bullet entirely.

```suggestion
- **Is this extensible enough?** ~~The above issue discusses adding arguments as
  part of this specification - we haven't touched on this here in order to keep
  this RFC small, but we may wish to consider this in the future (e.g.
  `Query.searchBusiness:name`).~~
```

Reply:
ack - it's under the sercion "Answered questions" which implies it's old news, but i'll just remove this point entirely as you suggest, since i guess it's superfluous as it's now explicitly now covered in the spec